packages/nixos/gpu.nix

```diff
@@ -123,7 +123,6 @@ in
     # instead.
 
     # Configure the persistenced for use with CC GPUs (e.g. H100).
-    # TODO(msanft): This needs to be adjusted for non-CC-GPUs.
```

Comment:
I'd say we should at least leave a note here along the lines of
* This setup does not work for normal GPUs.
* Here are resources for how it _might be possible_ to change this for normal GPUs.